src/components/Login/ForgotPassword/index.js

```diff
@@ -35,7 +35,9 @@ const ForgotPassword = ({ classes, email, error, handleForgotPassword, history,
   return (
     <Grid container spacing={24} className={classes.forgotPwdMainContainer}>
       <Grid item xs={12} sm={12} md={12} lg={12} className={classes.forgotPwdContent}>
-        <h2>Forgot your pasword?</h2>
+        <h2>
+          {window.location.pathname.includes(Routes.RESET_PASSWORD) ? "Reset your password" : "Forgot your pasword?"}
```

Comment:
replace window.location with the react-router location. It is received a props by default